packages/xstate-lit/README.md

```diff
@@ -0,0 +1,147 @@
+# @xstate/lit
+
+The [@xstate/lit](https://github.com/lit/lit) package contains a [Reactive Controller](https://lit.dev/docs/composition/controllers/) for using XState with Lit.
+
+- [Read the full documentation in the XState docs](https://stately.ai/docs/xstate-lit/).
+- [Read our contribution guidelines](https://github.com/statelyai/xstate/blob/main/CONTRIBUTING.md).
+
+## Quick Start
+
+1. Install `xstate` and `@xstate/lit`:
+
+```bash
+npm i xstate @xstate/lit
+```
+
+**Via CDN**
+
+```html
+<script src="https://unpkg.com/@xstate/lit/dist/xstate-lit.esm.js"></script>
+```
+
+2. Import the `UseMachine` Lit controller:
+
+**`new UseMachine(this, {machine, options?, subscriptionProperty?)`**
+
+```js
+import { html, LitElement } from 'lit';
+import { customElement, state } from 'lit/decorators.js';
+import { createBrowserInspector } from '@statelyai/inspect';
+import { createMachine } from 'xstate';
+import { UseMachine } from '@xstate/lit';
+
+const { inspect } = createBrowserInspector({
+  // Comment out the line below to start the inspector
+  autoStart: false,
+});
+
+const toggleMachine = createMachine({
+  id: 'toggle',
+  initial: 'inactive',
+  states: {
+    inactive: {
+      on: { TOGGLE: 'active' },
+    },
+    active: {
+      on: { TOGGLE: 'inactive' },
+    },
+  },
+});
+
+@customElement('toggle-component')
+export class ToggleComponent extends LitElement {
+  toggleController: UseMachine<typeof toggleMachine>;
+
+  @state()
+  protected _xstate: { [k: string]: unknown } = {};
+
+  constructor() {
+    super();
+    this.toggleController = new UseMachine(this, {
+      machine: toggleMachine,
+      options: { inspect },
+      subscriptionProperty: '_xstate',
```

Comment:
couldnt we just have a subscription callback for people to hook into and do their thing? I'm not used to Lit so maybe that's how things are done in that ecosystem but this option feels clumsy to me and makes types very loosely coupled (to the point that this doesn't really deliver good type-safety)

Reply:
Hi @Andarist,
I have replaced 'subscriptionProperty' as you suggest with a 'callback', this makes it much more aligned and also with [lit --> observers](https://github.com/lit/lit/tree/main/packages/labs/observers)